src/mem/ruby/protocol/MOESI_AMD_Base-CorePair.sm

```diff
@@ -2427,19 +2426,16 @@ machine(MachineType:CorePair, "CP-like Core Coherence")
   }
 
   transition(MO_I, NB_AckWB, I) {L2TagArrayWrite} {
-    wb_data;
```

Comment:
Remove the wb_data action since these are the only 4 cases it is used

Reply:
@NSurawar : has this been addressed?